### ISSUE TITLE
More jsonlines examples

### ISSUE THREAD

**Issue by egilron** [vdobrovolskii/wl-coref#17]
Thank you for the great research and repo! I want to train a model for Norwegian coref, and think it is best to produce a format similar to the output of `convert_to_jsonlines.py`. Maybe to the output of `convert_to_heads.py`. I read the explaination in [issue #6](https://github.com/vdobrovolskii/wl-coref/issues/6). Could you please share a couple of json lines for each of them, fully populated? I struggle with the registration for OntoNotes, and do not really need the data, only some examples. The `sample_input.jsonlines` has three fields, while I was hoping for an exaple with all fields: document_id, cased_words, sent_id, part_id, speaker, pos, deprel, head, clusters. 

I am in particular concerned about the text formatting of my pos and deprel information, it probably needs some convertion.

**Comment by vdobrovolskii** [comment]
Hi! Thanks for your interest!

I'm sharing it right here instead of uploading somewhere so that it doesn't get lost.
Here's a jsonline `from convert_to_jsonlines.py` (I replaced all the words to avoid copyright issues):

    {'document_id': 'bc/cctv/00/cctv_0005', 'cased_words': ['WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD'], 'sent_id': [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 4, 4, 4, 4, 4, 5, 5, 5, 5, 5, 5, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 7, 7, 7, 7, 7], 'part_id': 0, 'speaker': ['speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li'], 'pos': [':', 'RB', ',', 'PRP', 'VBD', 'RB', 'VBN', 'IN', 'IN', 'DT', 'JJ', 'JJ', 'NNS', '.', 'TO', 'VB', 'PRP$', 'NN', ',', 'DT', 'JJ', 'NNS', 'JJ', 'NN', 'VBZ', 'DT', 'NN', 'NN', 'IN', 'DT', 'NN', 'WDT', 'VBZ', 'VBN', 'VBN', '.', 'PRP', 'VBZ', 'NN', 'TO', 'VB', 'IN', 'DT', 'NN', 'NN', 'MD', 'RB', 'VB', 'NNS', ',', 'CC', 'IN', 'DT', 'NNS', 'WDT', 'VBP', 'IN', 'DT', 'NN', 'NN', 'MD', 'VB', 'RB', 'VBN', '.', 'NNP', 'NNS', ',', 'DT', 'NNP', 'NNP', 'NN', 'MD', 'VB', 'RB', '.', 'DT', 'VBZ', 'JJ', 'NNP', '.', 'VBP', 'PRP', 'NN', 'IN', 'VBG', '.', 'VBG', 'RP', 'VBZ', 'DT', 'NNP', 'NNP', 'NN', 'VBN', 'IN', 'NNP', 'NNP', '.', 'UH', ',', 'NN', 'NNS', '.'], 'deprel': ['punct', 'advmod', 'punct', 'nsubjpass', 'auxpass', 'advmod', 'root', 'prep', 'prep', 'det', 'amod', 'amod', 'pobj', 'punct', 'aux', 'advcl', 'poss', 'dobj', 'punct', 'det', 'amod', 'nn', 'amod', 'nsubj', 'root', 'det', 'nn', 'dobj', 'prep', 'det', 'pobj', 'nsubjpass', 'aux', 'auxpass', 'rcmod', 'punct', 'nsubj', 'root', 'dobj', 'aux', 'xcomp', 'mark', 'det', 'nn', 'nsubj', 'aux', 'advmod', 'ccomp', 'dobj', 'punct', 'cc', 'mark', 'det', 'nsubjpass', 'nsubj', 'rcmod', 'prep', 'det', 'nn', 'pobj', 'aux', 'auxpass', 'advmod', 'conj', 'punct', 'nn', 'nsubj', 'punct', 'det', 'nn', 'nn', 'nsubj', 'aux', 'root', 'advmod', 'punct', 'nsubj', 'cop', 'amod', 'root', 'punct', 'root', 'iobj', 'dobj', 'prep', 'pcomp', 'punct', 'root', 'prt', 'aux', 'det', 'nn', 'nn', 'nsubj', 'partmod', 'prep', 'nn', 'pobj', 'punct', 'discourse', 'punct', 'nn', 'root', 'punct'], 'head': [6, 6, 6, 6, 6, 6, None, 6, 6, 12, 12, 12, 8, 6, 15, 24, 17, 15, 24, 23, 23, 23, 23, 24, None, 27, 27, 24, 24, 30, 28, 34, 34, 34, 30, 24, 37, None, 37, 40, 37, 47, 44, 44, 47, 47, 47, 40, 47, 47, 47, 63, 53, 63, 55, 53, 55, 59, 59, 56, 63, 63, 63, 47, 37, 66, 73, 73, 71, 70, 71, 73, 73, None, 73, 73, 79, 79, 79, None, 79, None, 81, 81, 81, 84, 81, None, 87, 87, 93, 92, 93, 87, 93, 94, 97, 95, 87, 102, 102, 102, None, 102], 'clusters': [[[16, 17], [19, 24]], [[25, 28], [42, 45], [57, 60]], [[82, 83], [83, 84]]]}

And here's the same one from `convert_to_heads.py`:

    {'document_id': 'bc/cctv/00/cctv_0005', 'cased_words': ['WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD', 'WORD'], 'sent_id': [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 4, 4, 4, 4, 4, 5, 5, 5, 5, 5, 5, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 7, 7, 7, 7, 7], 'part_id': 0, 'speaker': ['speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'speaker#1', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li', 'Xu_li'], 'pos': [':', 'RB', ',', 'PRP', 'VBD', 'RB', 'VBN', 'IN', 'IN', 'DT', 'JJ', 'JJ', 'NNS', '.', 'TO', 'VB', 'PRP$', 'NN', ',', 'DT', 'JJ', 'NNS', 'JJ', 'NN', 'VBZ', 'DT', 'NN', 'NN', 'IN', 'DT', 'NN', 'WDT', 'VBZ', 'VBN', 'VBN', '.', 'PRP', 'VBZ', 'NN', 'TO', 'VB', 'IN', 'DT', 'NN', 'NN', 'MD', 'RB', 'VB', 'NNS', ',', 'CC', 'IN', 'DT', 'NNS', 'WDT', 'VBP', 'IN', 'DT', 'NN', 'NN', 'MD', 'VB', 'RB', 'VBN', '.', 'NNP', 'NNS', ',', 'DT', 'NNP', 'NNP', 'NN', 'MD', 'VB', 'RB', '.', 'DT', 'VBZ', 'JJ', 'NNP', '.', 'VBP', 'PRP', 'NN', 'IN', 'VBG', '.', 'VBG', 'RP', 'VBZ', 'DT', 'NNP', 'NNP', 'NN', 'VBN', 'IN', 'NNP', 'NNP', '.', 'UH', ',', 'NN', 'NNS', '.'], 'deprel': ['punct', 'advmod', 'punct', 'nsubjpass', 'auxpass', 'advmod', 'root', 'prep', 'prep', 'det', 'amod', 'amod', 'pobj', 'punct', 'aux', 'advcl', 'poss', 'dobj', 'punct', 'det', 'amod', 'nn', 'amod', 'nsubj', 'root', 'det', 'nn', 'dobj', 'prep', 'det', 'pobj', 'nsubjpass', 'aux', 'auxpass', 'rcmod', 'punct', 'nsubj', 'root', 'dobj', 'aux', 'xcomp', 'mark', 'det', 'nn', 'nsubj', 'aux', 'advmod', 'ccomp', 'dobj', 'punct', 'cc', 'mark', 'det', 'nsubjpass', 'nsubj', 'rcmod', 'prep', 'det', 'nn', 'pobj', 'aux', 'auxpass', 'advmod', 'conj', 'punct', 'nn', 'nsubj', 'punct', 'det', 'nn', 'nn', 'nsubj', 'aux', 'root', 'advmod', 'punct', 'nsubj', 'cop', 'amod', 'root', 'punct', 'root', 'iobj', 'dobj', 'prep', 'pcomp', 'punct', 'root', 'prt', 'aux', 'det', 'nn', 'nn', 'nsubj', 'partmod', 'prep', 'nn', 'pobj', 'punct', 'discourse', 'punct', 'nn', 'root', 'punct'], 'head': [6, 6, 6, 6, 6, 6, None, 6, 6, 12, 12, 12, 8, 6, 15, 24, 17, 15, 24, 23, 23, 23, 23, 24, None, 27, 27, 24, 24, 30, 28, 34, 34, 34, 30, 24, 37, None, 37, 40, 37, 47, 44, 44, 47, 47, 47, 40, 47, 47, 47, 63, 53, 63, 55, 53, 55, 59, 59, 56, 63, 63, 63, 47, 37, 66, 73, 73, 71, 70, 71, 73, 73, None, 73, 73, 79, 79, 79, None, 79, None, 81, 81, 81, 84, 81, None, 87, 87, 93, 92, 93, 87, 93, 94, 97, 95, 87, 102, 102, 102, None, 102], 'head2span': [[16, 16, 17], [23, 19, 24], [27, 25, 28], [44, 42, 45], [59, 57, 60], [82, 82, 83], [83, 83, 84]], 'word_clusters': [[16, 23], [27, 44, 59], [82, 83]], 'span_clusters': [[[16, 17], [19, 24]], [[25, 28], [42, 45], [57, 60]], [[82, 83], [83, 84]]]}

Please, note that not every key is used, however. For instance, `pos` and `deprel` are never used, they are there for legacy reasons and you can safely ignore them, `head` is only used by `convert_to_heads.py` and not in the actual model.

Hope this helps!

Good luck with your research!

**Comment by egilron** [comment]
Great! Thank you. Will let you know if I succeed.

**Comment by egilron** [comment]
Thank you again for the wl-coref work. Yes, I have now trained wl-coref on new English data, the litbank coref annotations. The setup is in [litbank4wl-coref](https://github.com/egilron/litbank4wl-coref). By running the scripts there, one can get English example data without any copyright and registration issues. I did not parse the sentences to get the head of each word, though, since it is only a toy project before doing Norwegian coref.
